### ISSUE TITLE
Comprehensive contrast list to consider and corresponding dataset

### ISSUE THREAD

**Issue by BreziTasbi** [ivadomed/Bidsification#1]
In order to determine the contrast classifier shape and extend, I need to know the comprehensive list of contrast that the model should know about. 
Then, considering that the model should be as robust as possible. I would like to know which datasets I could use in order to collect an as diverse as possible dataset.
Thanks a lot

**Comment by jcohenadad** [comment]
how about starting from all data present on git-annex? Maybe @NathanMolinier you have additional ideas

**Comment by NathanMolinier** [comment]
Yes good idea, you should start by going through all our datasets to identify the contrasts that we have that correspond to 3D images. I have done something similar before, so you could start by looking this:
- https://docs.google.com/presentation/d/1IxHPy974hfwCkcaa6-2ogrxk4imOsM-jQgYecQ4wbdE/edit#slide=id.p 
- https://github.com/ivadomed/utilities/blob/nm/managing_data/scripts/analyze_data.py
